gr-blocks/include/gnuradio/blocks/correctiq.h

```diff
@@ -0,0 +1,44 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef INCLUDED_CORRECTIQ_CORRECTIQ_H
+#define INCLUDED_CORRECTIQ_CORRECTIQ_H
+
+#include <gnuradio/blocks/api.h>
+#include <gnuradio/sync_block.h>
+
+namespace gr {
+namespace blocks {
+
+/*!
+ * \brief <+description of block+>
```

Comment:
:) I love this commit so far. Could you copy & paste the content from the YAML's doc field here?